user-guide/Standard_Apps/SRM/SRM_migrating/SRM_migrating_booking_manager_full_configuration.md

```diff
@@ -0,0 +1,121 @@
+---
+uid: SRM_migrating_booking_manager_full_configuration
+---
+
+# Migrating a complete Booking Manager configuration
+
+The SRM framework allows you to export all artifacts of a Booking Manager application to facilitate deployment. You can then add the configuration to a Companion Files repository so it can be included in an install package, as detailed below.
+
+This functionality is available since version 1.2.24 of the SRM framework.
+
+Consider deploying the *Ziine SatUplink* booking application.
```

Comment:
I think this needs a bit more context. Why should this be deployed, and how (i.e. where to export this or find the export package)? Also, maybe this could become a separate note block?

Reply:
Maybe this phrase covers it better:
> Imagine you'd like to deploy the booking manager for the *Ziine SatUplink* virtual platform.

Not sure if that's suitable for docs.